libs/math/include/math/tensor.hpp

```diff
@@ -1063,7 +1063,12 @@ template <typename T, typename C>
 template <typename... Indices>
 typename Tensor<T, C>::Type &Tensor<T, C>::At(Indices... indices)
 {
-  assert(sizeof...(indices) == stride_.size());
+  if (sizeof...(indices) != stride_.size())
+  {
+    throw exceptions::WrongIndices(
+        "Wrong arguments quantity (" + std::to_string(stride_.size()) +
+        ") given to Tensor::At, expected: " + std::to_string(stride_.size()));
```

Comment:
the string should print the expected and actual size - it now prints actual twice.

Reply:
done